src/blueprints/tv2/action-factories/tv2-graphics-action-factory.ts

```diff
@@ -154,4 +195,307 @@ export class Tv2GraphicsActionFactory {
+        return PieceLifespan.STICKY_UNTIL_RUNDOWN_CHANGE
+    }
+  }
+
+  private createVizFullPilotTimelineObjects(
+    blueprintConfiguration: Tv2BlueprintConfiguration, fullscreenGraphicsData: Tv2FullscreenGraphicsManifestData): TimelineObject[] {
+    if (!blueprintConfiguration.studio.VizPilotGraphics.CleanFeedPrerollDuration) {
+      throw new MisconfigurationException(
+        'Missing configuration of \'VizPilotGraphics.CleanFeedPrerollDuration\' in settings.'
+      )
+    }
+    const videoMixerEnable: TimelineEnable = { start: blueprintConfiguration.studio.VizPilotGraphics.CutToMediaPlayer }
+    const sourceInput: number = blueprintConfiguration.studio.VizPilotGraphics.FullGraphicBackground
+    const upstreamEnable: TimelineEnable = { start: blueprintConfiguration.studio.VizPilotGraphics.CleanFeedPrerollDuration }
+    const downstreamKeyer: Tv2DownstreamKeyer = this.getDownstreamKeyerMatchingRole(blueprintConfiguration, Tv2DownstreamKeyerRole.FULL_GRAPHICS)
+    const priority: number = 0
+
+    return [
+      this.vizTimelineObjectFactory.createFullGraphicsTimelineObject(blueprintConfiguration, fullscreenGraphicsData),
+      this.videoMixerTimelineObjectFactory.createProgramTimelineObject(GRAPHICS_PROGRAM_ID, sourceInput, videoMixerEnable),
+      this.videoMixerTimelineObjectFactory.createCleanFeedTimelineObject(GRAPHICS_CLEAN_FEED_ID, sourceInput, videoMixerEnable),
+      this.videoMixerTimelineObjectFactory.createLookaheadTimelineObject(GRAPHICS_LOOKAHEAD_ID, sourceInput, videoMixerEnable),
+      this.videoMixerTimelineObjectFactory.createDownstreamKeyerTimelineObject(downstreamKeyer, true, priority),
+      this.videoMixerTimelineObjectFactory.createUpstreamKeyerTimelineObject(
+        downstreamKeyer,
+        upstreamEnable
+      ),
+      this.audioTimelineObjectFactory.createStudioMicrophonesUpTimelineObject(blueprintConfiguration)
+    ]
+  }
+
+  private createCasparCgFullscreenGraphicsAction(blueprintConfiguration: Tv2BlueprintConfiguration, graphicsData: Tv2FullscreenGraphicsManifestData): Tv2PartAction {
+    if (!blueprintConfiguration.studio.HTMLGraphics) {
+      throw new MisconfigurationException(
+        'Missing configuration of \'HTMLGraphics\' in settings. Make sure it exists, and contains a value for \'KeepAliveDuration\''
+      )
+    }
+
+    const partId: string = 'fullGraphicsPart'
+    const fullGraphicPiece: Tv2PieceInterface = this.createCasparCgFullGraphicsPieceInterface(partId, blueprintConfiguration, graphicsData)
+    return this.createFullGraphicsActionFromGraphicsData(
+      blueprintConfiguration.studio.HTMLGraphics.KeepAliveDuration,
+      partId,
+      fullGraphicPiece,
+      graphicsData
+    )
+  }
+
+  private createCasparCgFullGraphicsPieceInterface(partId: string, blueprintConfiguration: Tv2BlueprintConfiguration, fullscreenGraphicsData: Tv2FullscreenGraphicsManifestData): Tv2PieceInterface {
+    return this.createGraphicsPieceInterface({
+      id: 'fullGraphicPiece',
```

Comment:
`full` should be `fullscreen` and `Graphic` should be `Graphics`.

Reply:
Have updated a number of `full` to `fullscreen`, and added missing 's'.